src/ralph/settings/base.py

```diff
@@ -167,3 +168,7 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',  # noqa
     'PAGE_SIZE': 10,
 }
+
+
+IMEI_UNTIL_2003 = re.compile(r'^\d{6} *\d{2} *\d{6} *\d$')
```

Comment:
it's not allegro specific - it's international standard (https://en.wikipedia.org/wiki/International_Mobile_Station_Equipment_Identity#Structure_of_the_IMEI_and_IMEISV_.28IMEI_Software_Version.29), so IMO this should be placed in source code, not settings (it was placed in source code in old assets too: https://github.com/allegro/ralph_assets/blob/33073a541c7484ce294f385e14fd167424193e50/src/ralph_assets/forms.py#L373)